lib/govuk_app_config/govuk_healthcheck/redis.rb

```diff
@@ -0,0 +1,21 @@
+module GovukHealthcheck
+  class Redis
+    def name
+      :redis
+    end
+
+    def status
+      client = Redis.new
```

Comment:
```suggestion
      client = ::Redis.new
```

I think this'll need to be `::Redis` to avoid accidentally calling `GovukHealthcheck::Redis`. Pity the tests can't really catch this :( 

Similar to: https://github.com/alphagov/govuk_app_config/blob/50c5320f2fb4756f3bb988ae3bc77e4978b47dd0/lib/govuk_app_config/govuk_healthcheck/active_record.rb#L8

Reply:
👍